demo-template/template/template.yaml

```diff
@@ -0,0 +1,137 @@
+# yaml-language-server: $schema=https://json.schemastore.org/yamllint.json
+
+apiVersion: scaffolder.backstage.io/v1beta3
+kind: Template
+metadata:
+  name: demo-template
+  title: Demo Template
+  description: Software Template used to demo Scaffolder features
+spec:
+  owner: backstage/maintainers
+  type: website
+
+  parameters:
+    - title: Fill in some steps
+      required:
+        - name
+        - mode
+        - topics
+      properties:
+        name:
+          title: What is your Name?
+          description: Enter your name
+          type: string
+          ui:autofocus: true
+        mode:
+          title: What do you prefer Light or Dark mode?
+          description: Select the mode your prefer
+          type: string
+          ui:widget: radio
+          enum:
+            - light
+            - dark
+          enumNames:
+            - Light
+            - Dark
+        topics:
+          title: What Software Template topics are you interested in?
+          description: Select one ore more Software Template topics you are interested in
+          type: array
+          uniqueItems: true
+          ui:widget: checkboxes
+          items:
+            type: string
+            enum:
+              - writing-templates
+              - input-examples
+              - builtin-actions
+              - writing-custom-actions
+              - writing-custom-field-extensions
+              - adding-your-own-templates
+            enumNames:
+              - Writing Templates
+              - Input Examples
+              - Builtin Actions
+              - Writing Custom Actions
+              - Writing Custom Field Extensions
+              - Adding Your Own Templates
+
+    - title: Choose a location
+      required:
+        - repoUrl
+      properties:
+        repoUrl:
+          title: Enter an Owner and Repo Name
+          description: This will create a new repo in the location you provide using your GitHub credentials
+          type: string
+          ui:field: RepoUrlPicker
+          ui:options:
+            requestUserCredentials:
+              secretsKey: USER_OAUTH_TOKEN
+              additionalScopes:
+                github:
+                  - workflow
+                  - administration
```

Comment:
sounds severe - is it, and is there nothing lower that fits the bill?

Reply:
Ends up I didn't need these and was able to remove `additionalScopes` entirely 👍